docs/site/Authentication-component-action.md

```diff
@@ -0,0 +1,252 @@
+
+The default sequence which injects and invokes actions `findRoute`,
+`parseParams`, `invoke`, `send`, `reject` could be found in the
+[Todo example's sequence file](https://github.com/strongloop/loopback-next/blob/master/examples/todo/src/sequence.ts)
+
+To know more details of what each action does, click the code snippet below with
+more descriptions.
+
+<details>
+<summary>Click to view the details of the default sequence</summary>
+
+```ts
+export class DefaultSequence implements SequenceHandler {
+  /**
+   * Constructor: Injects findRoute, invokeMethod & logError
+   * methods as promises.
+   *
+   * @param {FindRoute} findRoute Finds the appropriate controller method,
+   *  spec and args for invocation (injected via SequenceActions.FIND_ROUTE).
+   * @param {ParseParams} parseParams The parameter parsing function (injected
+   * via SequenceActions.PARSE_PARAMS).
+   * @param {InvokeMethod} invoke Invokes the method specified by the route
+   * (injected via SequenceActions.INVOKE_METHOD).
+   * @param {Send} send The action to merge the invoke result with the response
+   * (injected via SequenceActions.SEND)
+   * @param {Reject} reject The action to take if the invoke returns a rejected
+   * promise result (injected via SequenceActions.REJECT).
+   */
+  constructor(
+    @inject(SequenceActions.FIND_ROUTE) protected findRoute: FindRoute,
+    @inject(SequenceActions.PARSE_PARAMS) protected parseParams: ParseParams,
+    @inject(SequenceActions.INVOKE_METHOD) protected invoke: InvokeMethod,
+    @inject(SequenceActions.SEND) public send: Send,
+    @inject(SequenceActions.REJECT) public reject: Reject,
+  ) {}
+
+  /**
+   * Runs the default sequence. Given a handler context (request and response),
+   * running the sequence will produce a response or an error.
+   *
+   * Default sequence executes these steps
+   *  - Finds the appropriate controller method, swagger spec
+   *    and args for invocation
+   *  - Parses HTTP request to get API argument list
+   *  - Invokes the API which is defined in the Application Controller
+   *  - Writes the result from API into the HTTP response
+   *  - Error is caught and logged using 'logError' if any of the above steps
+   *    in the sequence fails with an error.
+   *
+   * @param context The request context: HTTP request and response objects,
+   * per-request IoC container and more.
+   */
+  async handle(context: RequestContext): Promise<void> {
+    try {
+      const {request, response} = context;
+      const route = this.findRoute(request);
+      const args = await this.parseParams(request, route);
+      const result = await this.invoke(route, args);
+
+      debug('%s result -', route.describe(), result);
+      this.send(response, result);
+    } catch (error) {
+      this.reject(context, error);
+    }
+  }
+}
+```
+
+</details>
+
+By default, `authenticate` is **not** part of the sequence of actions, so you
```

Comment:
`authenticate` == the authenticate action?

Reply:
yep, seems the word is confusing, any better suggestion? maybe "authenticate"?

Reply:
👌 changed to "authenticate"

Reply:
👌 changed to `authenticate`